rakelib/artifacts.rake

```diff
@@ -54,20 +54,44 @@ namespace "artifact" do
     File.open(".bundle/config", "w") { }
   end
 
-  # locate the "gem "logstash-core" ..." line in Gemfile, and if the :path => "." option if specified
+  # locate the "gem "logstash-core" ..." line in Gemfile, and if the :path => "..." option if specified
   # build and install the local logstash-core gem otherwise just do nothing, bundler will deal with it.
   task "install-logstash-core" do
+    RE = /^\s*gem\s+["']logstash-core["'](?:\s*,\s*["'][^"^']+["'])?(?:\s*,\s*:path\s*=>\s*["']([^"^']+)["'])?/i
```

Comment:
is this just trying to match `gem "logstash-core", :path => "/path/to/dir"`?
I'd suggest something as `LS_CORE_PATH_RE = /gem\s+["']logstash-core["'].+?:path/`
this is enough to catch the line and simpler to read